tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -237,14 +240,20 @@ install_cc_image() {
 	local component="rootfs-image"
 	local root_hash_vanilla="root_hash_vanilla.txt"
 	local root_hash_tdx=""
+	local initramfs_version=""
 	if [ -n "${tee}" ]; then
+		jenkins="${jenkins_url}/job/kata-containers-2.0-rootfs-image-${tee}-cc-$(uname -m)/${cached_artifacts_path}"
 		if [ "${tee}" == "tdx" ]; then
-			jenkins="${jenkins_url}/job/kata-containers-2.0-rootfs-image-${tee}-cc-$(uname -m)/${cached_artifacts_path}"
 			component="${tee}-rootfs-image"
 			root_hash_vanilla=""
 			root_hash_tdx="root_hash_${tee}.txt"
 		fi
-	fi
+		if [ "${tee}" == "sev" ]; then
+			component="${tee}-rootfs-initrd"
+			root_hash_vanilla=""
+			initramfs_version="-$(get_initramfs_image_name)"
```

Comment:
We don't want to use `$(get_initramfs_image_name)` here.  What we actually are looking for is the last commit that changed the scripts related to building the initramfs, no?

Also, please, do not add the `-` to the name, as it gets confusing when reading it.

Reply:
Fixed the name.